index.bs

```diff
@@ -3068,9 +3068,76 @@ To <dfn noexport>set an OS-support header</dfn> given a [=header list=]
 
 Issue: Fill in this section.
 
-# Privacy consideration # {#privacy-considerations}
+# Privacy considerations # {#privacy-considerations}
 
 The [=attribution caches=] contain data about a user's web activity. As such,
 the user agent MAY expose controls that allow the user to delete data from them.
 
-Issue: Fill in this section.
+Issue: Discuss general principles behind design decisions for the API.
+
+## Reporting-delay concerns ## {#reporting-delay-concerns}
+
+Sending reports some time after attribution occurs enables side-channel leakage
+in some situations.
+
+### Cross-network reporting-origin leakage ### {#cross-network-reporting-origin-leakage}
+
+A report may be *stored* while the browser is connected to one network but
+*sent* while the browser is connected to a different network, potentially
+enabling cross-network leakage of the reporting origin.
+
+Example: A user runs the browser with a particular browsing profile on their
+home network. An attribution report with a particular reporting origin is stored
+with a scheduled report time in the future. After the scheduled report time is
+reached, the user runs the browser with the same browsing profile on their
+employer's network, at which point the browser sends the report to the reporting
+origin. Although the report itself may be sent over HTTPS, the reporting origin
+may be visible to the network administrator via DNS. Some reporting origins may
```

Comment:
Or via the TLS client hello, which can be mitigated with ECH:
https://datatracker.ietf.org/doc/html/draft-ietf-tls-esni